base/database/setup.go

```diff
@@ -1,5 +1,74 @@
 package database
 
+import (
+	"app/base/utils"
+	"database/sql"
+	"fmt"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+type PostgreSQLConfig struct {
+	Host        string
+	Port        int
+	User        string
+	Database    string
+	Password    string
+	SSLMode     string
+	SSLRootCert string
+	Debug       bool
+}
+
 func ConfigureDatabase() {
+	dbConfig := loadDbConfig()
+	sqlDb := openConn(dbConfig)
+	verifyConn(sqlDb)
+}
+
+func loadDbConfig() *PostgreSQLConfig {
+	config := PostgreSQLConfig{
```

Comment:
Seems good for now, but we should also get values from clowder bit later.

Reply:
agreed, need to investigate how to do it in Go :D